### DISCUSSION TITLE
What artist do you like?

### DISCUSSION THREAD

**Discussion by takashi-jp** [NAIST-SE/AttractingContributors#807] [2023 Attraction Contributions]
I love rock music. But I want to know any categorize music. So please add your's lists of favorite artists to my README file.

Come to my fork:
https://github.com/takashi-jp/AttractingContributors-rockmusic

**Comment by kohei-ts** [comment]
I added my favorite rock group.
Could you tell me about your favorite rock genre?
https://github.com/kohei-ts/AttractingContributors-MusicGenre
